### PR TITLE
fix: AddOn failing due to issue with IsAddOnLoaded and LoadAddOn

### PR DESCRIPTION
**Pull request by sebbeh**

### CODE REVIEW

**Comment by Stanzilla** [comment]
> _ No description provided. _

Hey there, we need to be backwards compatible because of classic, can you add the fallbacks as well please?

**Comment by sebbeh** [comment]
> > _ No description provided. _
> 
> Hey there, we need to be backwards compatible because of classic, can you add the fallbacks as well please?

Yeah sure.

There is still an issue with some localisation in _kgPanelsConfig_ that needs to be adressed but I don't know how to access those in the repo.

**Comment by Stanzilla** [comment]
> > > _ No description provided. _
> > 
> > 
> > Hey there, we need to be backwards compatible because of classic, can you add the fallbacks as well please?
> 
> Yeah sure.
> 
> There is still an issue with some localisation in _kgPanelsConfig_ that needs to be adressed but I don't know how to access those in the repo.

Awesome, thanks! The localization is here https://www.wowace.com/projects/kg-panels/localization

**Comment by sebbeh** [comment]
> > > > _ No description provided. _
> > > 
> > > 
> > > Hey there, we need to be backwards compatible because of classic, can you add the fallbacks as well please?
> > 
> > 
> > Yeah sure.
> > There is still an issue with some localisation in _kgPanelsConfig_ that needs to be adressed but I don't know how to access those in the repo.
> 
> Awesome, thanks! The localization is here https://www.wowace.com/projects/kg-panels/localization

Couldn't figure out how to fix the typo in the localisation key on WowAce but I added it back in the repo just to make it work for now at least.

**Comment by Stanzilla** [comment]
> > > > > _ No description provided. _
> > > > 
> > > > 
> > > > Hey there, we need to be backwards compatible because of classic, can you add the fallbacks as well please?
> > > 
> > > 
> > > Yeah sure.
> > > There is still an issue with some localisation in _kgPanelsConfig_ that needs to be adressed but I don't know how to access those in the repo.
> > 
> > 
> > Awesome, thanks! The localization is here [wowace.com/projects/kg-panels/localization](https://www.wowace.com/projects/kg-panels/localization)
> 
> Couldn't figure out how to fix the typo in the localisation key on WowAce but I added it back in the repo just to make it work for now at least.

What is the issue exactly?

File: kgPanels.lua [hunk 1]
```diff
@@ -178,7 +178,8 @@ local emptyLayout = {}
 local strmatch = string.match
 local tonumber = tonumber
 local ipairs = ipairs
-local IsAddOnLoaded = IsAddOnLoaded
+local IsAddOnLoaded = C_AddOns.IsAddOnLoaded or IsAddOnLoaded
+local LoadAddOn = C_AddOns.LoadAddOn or LoadAddOn
```
Comment (Stanzilla) [inline kgPanels.lua]:
```suggestion
local IsAddOnLoaded = C_AddOns and C_AddOns.IsAddOnLoaded or IsAddOnLoaded
local LoadAddOn = C_AddOns and C_AddOns.LoadAddOn or LoadAddOn
```

File: kgPanelsConfig/PanelHelper.lua [hunk 2]
```diff
@@ -1274,7 +1274,7 @@ function kgPanelsConfig:CreatePanelMenu(panelName, panelData, isDefault)
 							xOffset = {
 								type = "input",
 								name = L["X Offset"],
-								desc = L["X offset from center. NOTE: positive for up, negative for down."],
+								desc = L["X offset from center. NOTE: postive for up, negative for down."],
```
Comment (sebbeh) [inline kgPanelsConfig/PanelHelper.lua:1277]: These localisation keys were spell-corrected but in the actual localisation tables the key still has the spelling misstake and therefore it cannot match the keys. The actual value will still be correct though.

Comment (sebbeh) [reply kgPanelsConfig/PanelHelper.lua:1277]: From `kgPanels-enUS.lua`
`L["X offset from center. NOTE: postive for up, negative for down."] = "X offset from center. NOTE: Positive for up, negative for down."`

File: kgPanels.lua (hunk 1 above)
Comment (sebbeh) [reply kgPanels.lua]: Nice catch, fixed!

**Comment by Stanzilla** [comment]
thanks!

**Pull request merged**